Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -2,12 +2,13 @@
 using UnityEngine;
 namespace Mirror
 {
-    public class TelepathyTransport : TransportLayer
+    public class TelepathyTransport : MonoBehaviour, ITransport
```

Comment:
Add a ToString() that returns:
"Telepathy Server {localaddress}: {localport}" if server and
"Telepathy Connected to {remoteaddress}: {remoteport}" if client

That way it shows in the NetworkManagerHUD

Reply:
ip is only saved in networkmanager at the moment. then telepathytransport either has to access networkmanager (kind of an unnecessary dependency) or it has to cache the ip that is passed to startclient - which is kinda pointless since .ToString is only called in HUD if server.active

Reply:
Neither.   This is the ToString I have in the 2018 branch:

```cs
 public override string ToString()
        {
            if (IsConnected )
            {
                return $"TCP connected to {client.Client.RemoteEndPoint}";
            }
            if (Connecting)
            {
                return $"TCP connecting to {client.Client.RemoteEndPoint}";
            }
            return "";
        }
```

client is just the TcpClient,  the socket itself has the RemoteEndPoint which has ip and port